Changelog documentation

alainamstutz opened the pull request:
Hi Tom,
I updated the changelog describing the changes between v0.0.1 - v0.05. 

Please, let me know if this should be more detailed. Happy to re-review any step.

Thank you very much! 
Best,
-Alain

alainamstutz committed:
Update CHANGELOG.md

alainamstutz committed:
Updated changelog annotation

remlapmot committed:
Attempt to fix GHA workflow

remlapmot reviewed (approved):
Looks good Alain

I added a small commit to try and solve the write permissions of the tagging action - however, I'm not sure if this will fix it. Maybe wait before Iain has replied and you've then toggled the Actions settings off then back on before merging this. (Otherwise manually pushing up tags is working at least).

Tom

alainamstutz committed:
Update main.yml

Use concurrency to ensure that only a single job using the same concurrency group will run at a time

alainamstutz commented:
Thanks, Tom! 
Looking at [Iain's comment](https://github.com/opensafely-actions/diabetes-algo/issues/13), I think `concurrency` does the trick. I added it, not sure if we are overdoing it now having both solutions in main.yml. Shall I try?

alainamstutz committed:
Update CHANGELOG.md

remlapmot reviewed (approved):
Happy for you try both Iain and my potential fixes.

I don't think Iain's fix is for our tagging fail.

Anyway you may as well merge this - and if tagging fails just push tags manually until you've got permission to toggle GHA off/on and ensure it says write permissions in repo settings.
